### ISSUE TITLE
api pr0plem

### ISSUE THREAD

**Issue by wangm-word** [open-mmlab/mmpose#811]
![image](https://user-images.githubusercontent.com/72743278/126928399-38f0194d-d2fd-4308-87bd-fc09ffa4ba3f.png)
thank you very much ! can you tell what question about this

**Comment by wangm-word** [comment]
@ly015 about config path

**Comment by ly015** [comment]
You need to import mmpose first, during which the models will be added to the registry. After that, the model can be built from the config.

**Comment by wangm-word** [comment]
than you very much ！But I am still a bit confused. i have imported mmpose, the config-path in init_pose_model() is wrong.  i want to inference a img and show it  use lite_hrnet.can you give a example about hou to use the api? thank you again
@ly015

**Comment by wangm-word** [comment]
![image](https://user-images.githubusercontent.com/72743278/126930672-854c75c3-fd03-47ba-b196-50a056d513ac.png)

**Comment by jin-s13** [comment]
https://mmpose.readthedocs.io/en/latest/demo.html

**Comment by ly015** [comment]
If you have the ground truth bounding boxes of your image, try:
```python
python demo/top_down_img_demo.py \
    configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/litehrnet_30_coco_256x192.py \
    https://download.openmmlab.com/mmpose/top_down/litehrnet/litehrnet30_coco_256x192-4176555b_20210626.pth \
    --img-root YOUR_IMAGE_ROOT --json-file YOUR_JSON_FILE \
    --out-img-root vis_results
```

If the bounding box information is not available, you can use mmdet to detect the bounding box:
```python
python demo/top_down_img_demo_with_mmdet.py \
    demo/mmdetection_cfg/faster_rcnn_r50_fpn_coco.py \ 
 https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
    configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/litehrnet_30_coco_256x192.py \
    https://download.openmmlab.com/mmpose/top_down/litehrnet/litehrnet30_coco_256x192-4176555b_20210626.pth \
    --img-root YOUR_IMAGE_ROOT \
    --img YOUR_IMAGE_FILENAME \
    --out-img-root vis_results
```

**Comment by wangm-word** [comment]
than you !when i use your method, it appaer new error,like this
@ly015 
![image](https://user-images.githubusercontent.com/72743278/126959630-030f8eac-2072-4787-bd01-dbfb583a5e22.png)

**Comment by ly015** [comment]
Please check:
1. What is your mmpose version?
2. Have you installed mmpose following the [document](https://github.com/open-mmlab/mmpose/blob/master/docs/install.md)
3. Have you modified the demo script?

**Comment by wangm-word** [comment]
Thank you for answering my question！！
mmpose is 0.16.0
torch is 1.7.1
cuda is 10.1
mmdet is 2.14.0
i have installed mmpose ,but this error still appeared
@ly015

**Comment by ly015** [comment]
Thanks for the information. Have you modified the demo script or any other part of mmpose?

**Comment by wangm-word** [comment]
no,only liter-hrnet's config file and .pth file is deferent from your example,this tow file come from official ，https://github.com/HRNet/Lite-HRNet
@ly015

**Comment by jin-s13** [comment]
You may use the master version of MMPose, which has already supported Lite-HRNet.

**Comment by ly015** [comment]
Yes, LiteHRNet was not implemented in v0.16.0, thus the model could not be built from the config file (Also, the official config file is not guaranteed to work in mmpose). Please use the latest master which has supported LiteHRNet.
